### ISSUE TITLE
Missing cmake macros in LCIO package config

### ISSUE THREAD

**Issue by JeremyMcCormick** [slaclab/slic#39]
The LCIO package config files reference some cmake macros that exist in the LCIO source tree but are not installed.  This can be remedied by adding the directory where they are located to the `CMAKE_MODULE_PATH` so that they can be resolved.

**Comment by JeremyMcCormick** [comment]
If you manage your LCIO installation using SLIC itself, then you can get a clean build e.g.

```
cmake -DLCIO_TAG=v02-07-04 -DINSTALL_DEPENDENCIES=ON -DGEANT4_INSTALL_DATA=ON -DCMAKE_INSTALL_PREFIX=/u/ey/jeremym/hps-dev/slic/install ..
```

Then you can build as follows:

```
make # builds dependencies
cmake .. # resolves dependencies
make install # builds and installs slic
```

I have opened an issue on the LCIO bug tracker to see what they suggest to resolve this.

**Comment by JeremyMcCormick** [comment]
Fixed by #40.

Use simple `FindLCIO.cmake` macro instead of LCIO's built-in cmake config.
